drivers/ieee802154/ieee802154_kw41z.c

```diff
@@ -1020,6 +1020,12 @@ static void kw41z_iface_init(struct net_if *iface)
 	ieee802154_init(iface);
 }
 
+static int kw41z_configure(struct device *dev, enum ieee802154_config_type type,
```

Comment:
fix you commit title:  "drivers/ieee802154: Fix KW41Z driver fault"  and have a message explaining what was the issue.